scripts/test.sh

```diff
@@ -11,7 +11,9 @@ for crate in $crates; do
     echo "Testing crate: $crate..."
 
     # Always test the crate itself first, without any features added.
-    $CARGO ${CARGO_COMMAND} -p $crate
+    if [[ "$crate" != "utoipa-gen" ]]; then
+        $CARGO ${CARGO_COMMAND} -p $crate
+    fi
```

Comment:
So this was added so that we always check that it is possible to build/check/test a crate without any explicit features being added on the command line.

However, that concept does not easy work with utoipa-gen , so I have added an exception here to avoid that.

Reply:
Ah, True it does not since there are those mutually exclusive features which is unfortunate, but kind of hard to move out.